examples/03_connectivity/plot_seed_based_correlation.py

```diff
@@ -0,0 +1,151 @@
+"""
+Producing single subject maps of seed-based correlation
+========================================================
+
+This example shows how to produce seed-based correlation maps for a single subject based on resting-state fMRI scans.
+These maps depict the temporal correlation of a **seed region** with the **rest of the brain**.
```

Comment:
I would add here: that "This example is an advanced one that requires manipulating the data with numpy.

Note the difference between images, that lie in brain space space, and the numpy array, corresponding to the data inside the mask".